pkg/ebpf/ingress_node_firewall_loader.go

```diff
@@ -199,22 +198,18 @@ func (infc *IngNodeFwController) IngressNodeFwAttach(ifacesName ...string) error
 		}
 
 		// Attach the program.
-		l, err := link.AttachXDP(link.XDPOptions{
-			Program:   objs.IngressNodeFirewallProcess,
-			Interface: int(ifID),
-		})
-		if err != nil {
-			errors = append(errors, fmt.Errorf("could not attach XDP program: %s", err))
+		xdpProName := "../ebpf/bpf_bpfel.o"
```

Comment:
nit for readability:
~~~
xdpProgName or xdpProgramName
xdpProg or xdpProgram
~~~

Reply:
my IDE suggested the stupid name I was using :(